docs/persist.rst

```diff
@@ -55,7 +55,13 @@ Rasa NLU supports using `S3 <https://aws.amazon.com/s3/>`_ and
 
     If there is no container with the name ``AZURE_CONTAINER`` Rasa will create it.
 
-Models are gzipped before saving to cloud.
+Models are gzipped before saving to cloud. The gzipped file naming convention
+is `{PROJECT}___{MODEL_NAME}.tar.gz` and should be in the root folder of the storage service.
```

Comment:
```suggestion
is `{PROJECT}{MODEL_NAME}.tar.gz` and it is stored in the root folder of the storage service.
```

Reply:
Was this behavior changed in v0.14? We are using v0.13.7 so I tested on that.

Reply:
Line 96-100 in persistor.py: 
```
    @staticmethod
    def _project_prefix(project: Text) -> Text:

        p = project or RasaNLUModelConfig.DEFAULT_PROJECT_NAME
        return '{}___'.format(p)
```